### DISCUSSION TITLE
One time payout

### DISCUSSION THREAD

**Discussion by bugproof** [btcpayserver/btcpayserver#2940] [Technical Support] [answered]
Hi! Are one time payouts possible?

**Comment by bolatovumar** [comment]
@bugproof could you clarify what exactly you mean by "one time payout" here?

**Reply by bugproof** [reply]
Requesting a payout to a specific wallet address just 1 time.

**Comment by Kukks** [comment, marked as answer]
Lightning: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreLightningNodeApi_PayInvoice
On chain:  https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreOnChainWallets_CreateOnChainTransaction
